erpnext/hooks.py

```diff
@@ -279,7 +279,7 @@
 		'validate': ['erpnext.regional.india.utils.set_place_of_supply']
 	},
 	('Sales Invoice', 'Purchase Invoice'): {
-		'validate': ['erpnext.regional.india.utils.validate_document_name']
+		'validate': ['erpnext.regional.india.utils.validate_document_name', 'erpnext.regional.india.utils.update_taxable_values']
```

Comment:
Is this supposed to be calculated for Purchase Invoices? If yes, then we need to add the `taxable_value` field for the Purchase Invoice Item too.